web/css/main.css

```
@@ -246,41 +336,496 @@ body.pc-vertical-stretch {
+    @apply block;
+    @apply w-full;
+    @apply p-4;
+    @apply text-sm;
+    @apply font-mono;
+    @apply bg-white;
+    @apply text-pc-grey-950;
+    @apply outline-none;
+    @apply appearance-none;
+    @apply border-0;
+    @apply resize-none;
+    height: auto;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)